bin/dfx-software-sns-aggregator-install

```diff
@@ -54,7 +54,7 @@ test -e "$WASM_FILE" || {
 jq -e '.canisters.sns_aggregator' dfx.json >/dev/null || {
   echo "Adding sns_aggregator to dfx.json..."
   # TODO: Include API declaration .did files in release, so that we can get the candid.
-  jq -s '.[0] * .[1]' <(echo '{"canisters": { "sns_aggregator": { "type": "custom", "candid": "placeholder", "wasm": "'"${WASM_FILE}"'", "build": "true" }}}') dfx.json | sponge dfx.json
+  jq -s '.[0] * .[1]' <(echo '{"canisters": { "sns_aggregator": { "type": "custom", "candid": "candid/sns_aggregator.did", "wasm": "'"${WASM_FILE}"'", "build": "true" }}}') dfx.json | sponge dfx.json
```

Comment:
How is the aggregator.did populated?

Reply:
Lines 20-22 in `bin/dfx-sns-aggregator-wait`:
```
: Make sure the candid is available and of the right version.
DID_FILE="$(jq -r .canisters.sns_aggregator.candid dfx.json)"
dfx canister metadata sns_aggregator candid:service >"$DID_FILE"
```